apps/backend/.env.example

```
@@ -12,3 +12,10 @@ DB_CONNECTION_STRING=[db_client]://[db_username]:[db_user_password]@localhost:[d
 DB_DIALECT=pg
 DB_POOL_MIN=2
 DB_POOL_MAX=10
+
+#
+# JWT
+#
+SECRET=.
+EXPIRATION_TIME=24hr
+ALGORITHM=HS256
```

```suggestion
JWT_SECRET=.
JWT_EXPIRATION_TIME=24hr
JWT_ALGORITHM=HS256
```
Let's be more specific